.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: "Validate Gradle Wrapper"
+on: [push, pull_request]
+
+jobs:
+  validation:
+    name: "Gradle wrapper validation"
```

Comment:
nit: there seems to be a duplication that makes it show "Validate Gradle Wrapper / Gradle wrapper validation" in the checks. The job's name could really be "Validate" or something similar. Or maybe even "Gradle Wrapper" as the action's name and "validate" as the job's, so that we get "Gradle Wrapper / validate" in the checks.

/cc @simonbasle 
<img width="710" alt="Screenshot 2020-09-01 at 11 28 17 AM" src="https://user-images.githubusercontent.com/1050762/91832894-4227ec80-ec46-11ea-9036-d92130f2ba3a.png">

Reply:
good point, I'll rename it to `Gradle Wrapper / validate`